src/components/projecteditor/control/control.js

```diff
@@ -166,15 +169,15 @@ export default class Control extends Component {
      * if all windows successfully closed then remove project from explorer.
      */
     _closeProject = cb => {
-        const { router, closeAllPanels } = this.props;
+        const { router, closeTransactionsHistoryPanel } = this.props;
```

Comment:
Please revert this change. This was done in purpose as right now we only have one panel but soon we are going to have multiple panels (testing, preview, rewind, etc)